pytorch_lightning/utilities/distributed.py

```diff
@@ -25,6 +23,7 @@
 
 import pytorch_lightning as pl
 from pytorch_lightning.utilities.imports import _TORCH_GREATER_EQUAL_1_8, _TORCH_GREATER_EQUAL_1_9, _TPU_AVAILABLE
+from pytorch_lightning.utilities.rank_zero import rank_zero_debug, rank_zero_info, rank_zero_only  # noqa: F401
```

Comment:
are these going to stay here until some version?

don't we do deprecations like this in such cases:
```py
from new_module import func as func_new

def func(...):
    rank_zero_dep('func is moved to somewhere else...')
    func_new(...)
```